README.md

```diff
@@ -17,21 +17,22 @@ Examples
 Create a PDF file:
 
 ```python
-import trml2pdf
-print trml2pdf.parseString(file('file.rml','r').read())
+import trml2pdf.doc
+print trml2pdf.doc.parse_string(file('file.rml','r').read())
 ```
  
 If you are using this for Django you can dynamically create an .rml file with the template system and then render it.
 
 ```python
 from django.template.loader import get_template
 from django.template.context import Context
+import trml2pdf.doc
```

Comment:
Introducing incompatible changes, is it possible to re-export those functions from the root `trml2pdf` ?

Reply:
You are right.
Fixed in last commit